apps/client/src/components/Error.js

```diff
@@ -1,20 +1,23 @@
 import React from "react";
 import { Alert } from "@datapunt/asc-ui";
+import { ComponentWrapper } from "./Atoms";
 
 const Error = ({ heading, content, stack, children }) => (
-  <Alert
-    level="error"
-    heading={heading || `Er is een fout opgetreden.`}
-    content={content}
-    style={{ marginBottom: 25 }}
-  >
-    {children}
+  <ComponentWrapper>
+    <Alert
+      level="error"
+      heading={heading || `Er is een fout opgetreden.`}
+      content={content}
+      style={{ display: "block" }} // IE11 Fix
```

Comment:
It's easy to forget this IE11 fix everywhere we use Alert. Maybe we can have our own small wrapper around Alert in our atoms/components folder until (if ever) this bug is fixed in ASC.

Reply:
I was thinking not to do this, because sometimes you might need a `flexbox` Alert. But I guess it's better not needing to copy/paste all IE11 fixes for every new Alert.

FIxed